test/e2e/deploy_reaper_test.go

```diff
@@ -36,31 +38,21 @@ var (
 var _ = Describe("Deploy Reaper with Cassandra backend", func() {
 	Context("When a Cassandra cluster is deployed", func() {
 		Specify("Reaper is deployed", func() {
+			// Speed up cassandradatacenter_controller status checks
```

Comment:
When we initially discussed this scenario, I figured it would be a separate test. Do you think that would make more sense?

Reply:
Well, the reason I did in the main test was speed. Already the first test takes a lot of time to execute, so setting another with basically the same properties (other than one extra cluster) doubles the execution time. I'm not sure if there's value for two tests with only this change.

If we would get the cassdc startup to be faster, then adding multiple stages / tests would of course be better.